DentOS_Framework/DentOsTestbed/configuration/testbed_config/basic_dist1/testbed.json

```diff
@@ -11,6 +11,14 @@
                 "userName":"root",
                 "password":"onl"
             },
+            "cpuArchitecture": "arm",
+            "macAddress": "34:EF:B6:EC:26:BE",
+            "serialConsoleIp": "10.36.118.200",
+            "serialConsolePort": 7006,
+            "pduIp": "10.36.118.201",
+            "pduLogin": "dent",
+            "pduPassword": "dent123!",
+            "pduNumber": 6,
             "serialDev":"/dev/ttyUSB0",
```

Comment:
i would group them in 
pdu: {
ip
login
password
}

serial: {
device
ip
port....
}